src/components/table.yml

```diff
@@ -1,6 +0,0 @@
-prev:
-  en: ‹ back
-  de: ‹ zurück
-next:
-  en: › forward
-  de: › weiter
```

Comment:
These texts were also no longer the applied ones, since the AntD update, as the whole `itemRender` was not supported anymore via the table.

Reply:
:sweat_smile: